.github/workflows/maven.yml

```diff
@@ -122,5 +124,6 @@ jobs:
           FIRST_JAR="${ALL_JARS[0]}"
           cp "$FIRST_JAR" "deploy/dt-latest-$BRANCH_CLEAN.jar"
```

Comment:
In the datatools-build S3 bucket, this jar appears as `dt-latest-.jar` with the same date than `dt-53e045b.jar` of this branch. `dt-latest-.jar` should probably carry the branch name (see comment above).

Reply:
<img width="901" alt="Screen Shot 2021-07-01 at 11 53 31 AM" src="https://user-images.githubusercontent.com/86619099/124153908-f5c65c80-da62-11eb-9797-13474c218d4b.png">
This looks better right?